AppController/terminate.rb

```diff
@@ -78,6 +78,23 @@ def self.erase_appscale_state
     print "OK"
   end
 
+  # This functions ensure that the services AppScale started that have a
+  # PID in /var/run/appscale got terminated.
+  def self.ensure_services_are_stopped
+    Dir["/var/run/appscale/*.pid"].each { |pidfile|
```

Comment:
I'm a little concerned that very old pidfiles could contain PIDs that might have been reused by other processes.